pkgs/top-level/all-packages.nix

```diff
@@ -5648,7 +5648,9 @@ let
 
   gotty = goPackages.gotty.bin // { outputs = [ "bin" ]; };
 
-  gradle = callPackage ../development/tools/build-managers/gradle { };
+  gradleGen = callPackage ../development/tools/build-managers/gradle { };
+  gradle = self.gradleGen.gradle28;
+  gradle25 = self.gradleGen.gradle25;
```

Comment:
there is line
```
gradle25 = callPackage ../development/tools/build-managers/gradle/2.5.nix { };
```
right below. shouldn't it be removed?

You can also use `callPackages` idiom, as in https://github.com/NixOS/nixpkgs/pull/9834